jvm/src/test/scala/fm/common/TestClassUtil.scala

```diff
@@ -0,0 +1,153 @@
+  }
+
+  // classpathFileExists
+  /*
+  // Directories timestamps get changed every time they get moved to the new resource directory/jar file/etc,
+  // This was used to do a manual test for directories, but commenting out to get tests to pass
+  test("classpathLastModified - directories") {
+    // This is the project-relative path for the directory, and test assumes being ran in the project home
+    val f: File = new File("jvm/src/test/resources/test/classutil")
+    assert(f.isDirectory, s"$f must be a directory (is the working directory the project home?)")
+
+    testDirPaths.foreach{ ClassUtil.classpathLastModified(_) should equal(f.lastModified) }
+    testDirFiles.foreach{ ClassUtil.classpathLastModified(_) should equal(f.lastModified) }
+  }*/
+
+  test("classpathLastModified - files") {
+    val f: File = new File("jvm/src/test/resources/test/classutil/lorem-ipsum.txt")
+    assert(f.isFile, s"$f must be a file (is the working directory the project home?)")
+
+    testPaths.foreach { ClassUtil.classpathLastModified(_) should equal(f.lastModified) }
+    testFiles.foreach { ClassUtil.classpathLastModified(_) should equal(f.lastModified) }
+  }
+
+/*
+  test("def findAnnotatedClasses[T <: Annotation](basePackage: String, annotationClass: Class[T], classLoader: ClassLoader = defaultClassLoader): Set[Class[_]]") { }
+*/
+
+  test("findClassNames") {
+    ClassUtil.findClassNames("test.classutil") should equal(testClasses)
+  }
+
+  test("findClasspathFiles") {
+    // Includes recursive file(s)
+    val expectedFiles: Set[File] = {
+      testClasses.map{ _.replace(".", "/") + ".class" } ++ Set("test/classutil/lorem-ipsum.txt", "test/classutil/subdirectory/subfile.txt")
+    }.map{ new File(_) }
+
+    ClassUtil.findClasspathFiles("test.classutil") should equal (expectedFiles)
+  }
+
+  test("findImplementingClasses") {
+    //ClassUtil.findClasspathFiles(test.classutil.TestTrait) should equal(Seq.empty)
+  }
+
+  test("def findImplementingClasses[T](basePackage: String, clazz: Class[T], classLoader: ClassLoader = defaultClassLoader): Set[Class[_ <: T]]") { }
+
+/*
+  test("def findLoadedClass(cls: String, classLoader: ClassLoader = defaultClassLoader): Option[Class[_]]") { }
+
+  test("def isClassLoaded(cls: String, classLoader: ClassLoader = defaultClassLoader): Boolean") { }
+*/
+
+  test("def listClasspathFiles(basePackage: String, classLoader: ClassLoader = defaultClassLoader): Set[File]") {
+    // Does NOT include recursive file(s)
+    val updatedTestClasses: Set[String] = testClasses - "test.classutil.subpackage.TestSubPackageClass" // don't include subpackage class
+
+    val expectedFiles: Set[File] = {
+      updatedTestClasses.map{ _.replace(".", "/") + ".class" } ++ Set("test/classutil/lorem-ipsum.txt", "test/classutil/subpackage", "test/classutil/subdirectory")
+    }.map{ new File(_) }
+
+    ClassUtil.listClasspathFiles("test.classutil") should equal(expectedFiles)
```

Comment:
Can you add tests for `ClassUtil.listClasspathFiles("")` and `ClassUtil.listClasspathFiles("/")` ?